test/e2e/page-objects/pages/send/solana-confirm-tx-page.ts

```diff
@@ -55,7 +52,15 @@ class ConfirmSolanaTxPage {
    */
   async clickOnSend(): Promise<void> {
     const sendButton = await this.driver.findElement(this.sendButton);
+
     await sendButton.click();
```

Comment:
Could we use the clickElement method instead? Notice the clickElement already looks for a clickable element first, `findClickableElement` as well as has some guards implemented, so it's always better to use that instead of `.click()` which doesn't have any guards.

It's also more succint and we are not repeating checks.

```suggestion
    await this.driver.clickElement(this.sendButton);
```

Reply:
done!